src/Containers/Note/Note.js

```diff
@@ -1,80 +1,94 @@
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
 import { updateNote } from '../../actions';
-import { Route, Link } from 'react-router-dom';
-import { NoteForm } from '../NoteForm/NoteForm'; 
-import PropTypes from "prop-types";
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
 
 export class Note extends Component {
-  constructor({note}) {
-    super();
+  constructor(props) {
+    super(props);
     this.state = {
-      issues: note.issues || [],
-    }
+      issues: this.props.note.issues || []
```

Comment:
I changed this to be props instead of `{ note }` because I was receiving a warning in my console of ensuring to pass all props to super that were passed to the constructor. Then, in the render I just destructured note in the render